src/components/common/YourCompany/CorrectName/CorrectNameRequest.vue

```diff
@@ -235,22 +235,25 @@ export default class CorrectNameRequest extends Mixins(CommonMixin, NameRequestM
     const isNameEntityTypeDifferent = this.getEntityType !== nr.legalType
     const entityTypeOptions = this.getResource?.changeData?.entityTypeOptions
     const isValidConversionNameRequest = nr.request_action_cd === NrRequestActionCodes.CONVERSION &&
-          entityTypeOptions?.some(options => options.value === nr.legalType)
+      entityTypeOptions?.some(options => options.value === nr.legalType)
     return (isNameEntityTypeDifferent && !isValidConversionNameRequest)
   }
 
   /* Generate content of error depending on name request type. */
   nameRequestErrorText (nr: NrResponseIF): string {
+    let changeFrom = GetCorpFullDescription(nr.actions[0].filingName.split(' ')[0] as CorpTypeCd)
```

Comment:
It was nr.entity_type_cd here before, but it's not entity "changed from", actions is the only attribute contains changed from entity type, here is an example:
<img width="348" alt="image" src="https://github.com/bcgov/business-edit-ui/assets/16507223/84948914-b057-40a9-b73b-bf0d9ca33ef6">

Reply:
I don't think we should do this, this is fairly hacky, we should load in the correct entity_type_cd, so it populates the display. I can hunt down the data if needed.

Reply:
UX updated

Reply:
I agree with Travis: look at the Request Type Code.

See also: https://github.com/bcgov/bcrs-shared-components/blob/main/src/enums/name-request-codes.ts